README.md

```
@@ -1053,6 +1119,24 @@ readinessProbe:
   httpGet: { path: /readyz, port: 9394 }
 ```
 
+### Boot diagnostics banner
+
+`Supervisor#run` logs a one-block banner right after the heartbeat starts and before queues bootstrap, so a misconfigured deployment states its actual settings instead of forcing an operator to attach a console. Every line is `"[Pgbus] boot:"`-prefixed and renders cleanly under both the `:text` and `:json` log formatters:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)